packages/destination-actions/src/destinations/listrak/updateEmailContactProfileFields/index.ts

```diff
@@ -0,0 +1,94 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { makePostRequest } from '../listrak'
+import { RequestClient } from '@segment/actions-core'
+
+export type SegmentationFieldValue = {
+  segmentationFieldId: number
+  value: string
+}
+
+export type ContactSegmentationFieldValues = {
+  emailAddress: string
+  segmentationFieldValues: SegmentationFieldValue[]
+}
+
+type RequestsByListId = { [k: number]: ContactSegmentationFieldValues[] }
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Update Email Contact Profile Fields',
+  description: '',
+  fields: {
+    listId: {
+      label: 'List ID',
+      description:
+        'Identifier used to locate the list. Find this under Help & Support > API ID Information in https://admin.listrak.com.',
+      type: 'integer',
+      required: true
+    },
+    emailAddress: {
+      label: 'Email Address',
+      description: 'Email address of the contact.',
+      type: 'string',
+      format: 'email',
+      default: {
+        '@path': '$.context.traits.email'
+      },
+      required: true
+    },
+    profileFieldValues: {
+      label: 'Profile Field Values',
+      description:
+        'Add key value pairs to set one or more profile fields. The key is the profile field ID you want to set. Find this under Help & Support > API ID Information in https://admin.listrak.com. The value is the profile field value. (i.e. 1234 = on)',
+      type: 'object',
+      required: true,
+      defaultObjectUI: 'keyvalue:only'
+    }
```

Comment:
```suggestion
    },
    enable_batching: {
      type: 'boolean',
      label: 'Batch Data to Listrak',
      description:
        'When enabled, the action will use the LIstrak batch API. Which we recommend for...',
      required: true,
      default: false
  },
```

For batching you'll actually need to [add this extra field](https://github.com/segmentio/action-destinations/blob/main/README.md#batching-requests) `enable_batching`.

Please choose the default value and write a description that is relevant to your needs